internal/test/e2e/cluster.go

```diff
@@ -422,22 +422,17 @@ func newTestCluster(t *testing.T, opts testClusterOptions) *TestCluster {
 
 	// Fund the bus.
 	if funding {
-		cluster.MineBlocks(latestHardforkHeight)
+		cluster.MineBlocks(busCfg.Network.HardforkFoundation.Height + blocksPerDay + 10) // mine some extra blocks
```

Comment:
Is there a reason we can't mine to the latest hard fork height? I'd prefer for this to be precise since I can't think of anything requiring this to mine extra blocks.

Reply:
The blocks we mine have a production maturity delay of 144 blocks now. The 10 extra blocks I mine are just to build up some spendable money in the wallet so things like redistributing the wallet doesn't error out. Are you suggesting we mine right up until the hard fork height? I updated the code to match right up until latest hardfork height + the  maturity delay.

Reply:
I'd mine until hard fork + maturity delay then

Reply:
I'm not sure I fully understand the suggestion. I'm currently mining until hard fork + maturity delay. Adding a variable for that is a little odd because it would include the configurable `HardforkFoundation` height on the network? Maybe we can make it a method on the bus config but I'm struggling to find a good name for that too, it's not really the maturity delay nor the maturity height, it's just the height at which the first block matures. I added a comment for now to indicate we add `blocksPerDay` to the hardfork foundation height to make sure the first miner payout matured.